docs/src/appendix/cheatcodes/start_mock_call.md

```diff
@@ -0,0 +1,55 @@
+# `start_mock_call`
+
+> `fn start_mock_call<T, impl TSerde: serde::Serde<T>, impl TDestruct: Destruct<T>>(contract_address: ContractAddress, function_name: felt252, ret_data: T)`
+
+Mocks contract call to a `function_name` of a contract at the given address. A call to function `function_name` will return data provided in `ret_data` argument. 
+
+If contract is deployed at given address, mocked function won't be executed. Address with no contract can be mocked as well.
```

Comment:
`If contract is deployed at given address, mocked function won't be executed`
Wdym? I don't think this sentence is particularly helpful, it confused me at first

Reply:
There could be no contract deployed under the given address. If there is, and the call to the function was mocked, then this function will not be executed.

Reply:
Slightly updated in https://github.com/foundry-rs/starknet-foundry/pull/385/commits/dc2461084f0a10b3d556f4b6f51b177251552516